apps/nextjs-website/src/components/organisms/WebinarQuestionsTemplate/WebinarQuestionsTemplate.tsx

```diff
@@ -61,46 +53,31 @@ const WebinarQuestionsTemplate = ({
           endDateTime={webinar.endDateTime}
           webinarState={webinarState}
         >
-          <TableContainer component={Paper} sx={{ marginY: 2 }}>
-            <Table sx={{ minWidth: 650 }} aria-label='simple table'>
-              <TableHead>
-                <TableRow>
-                  <TableCell>{t('createdAt')}</TableCell>
-                  <TableCell>{t('text')}</TableCell>
-                  <TableCell></TableCell>
-                </TableRow>
-              </TableHead>
-              <TableBody>
-                {sortedQuestions.map((webinarQuestion) => (
-                  <WebinarQuestionRow
-                    key={webinarQuestion.id.createdAt.toISOString()}
-                    question={webinarQuestion}
-                    userName={userName}
-                    onHide={async (hide) =>
-                      await updateWebinarQuestion({
-                        id: webinarQuestion.id,
-                        updates: {
-                          hiddenBy: hide
-                            ? { operation: 'update', value: userName }
-                            : { operation: 'remove' },
-                        },
-                      })
-                    }
-                    onHighlight={async (highlight) =>
-                      await updateWebinarQuestion({
-                        id: webinarQuestion.id,
-                        updates: {
-                          highlightedBy: highlight
-                            ? { operation: 'update', value: userName }
-                            : { operation: 'remove' },
-                        },
-                      })
-                    }
-                  />
-                ))}
-              </TableBody>
-            </Table>
-          </TableContainer>
+          <Box
+            pb={4}
+            width={'100%'}
+            sx={{
+              display: 'flex',
+              flexDirection: { xs: 'column-reverse', md: 'row' },
+              alignContent: 'space-between',
+              gap: 4,
+            }}
+          >
+            <Grid key={'notHighlightedQuestions'} item xs={12} md={6}>
+              <WebinarQuestionsTable
+                userName={userName}
+                questions={notHighlightedQuestions}
+                title={'notHighlightedQuestions'}
```

Comment:
Why are you passing here the key that the `WebinarQuestionTable` should use to get the translated value?
Wouldn't be better and easier to pass the title instead?